src/frontends/paddle/src/op/pool3d.cpp

```diff
@@ -0,0 +1,334 @@
+            if ((input_d > 0) && (input_d + pad_begin[0] + pad_end[0] < kernel_d)) {
+                kernel_d = input_d + pad_begin[0] + pad_end[0];
+            }
+            if ((input_h > 0) && (input_h + pad_begin[1] + pad_end[1] < kernel_h)) {
+                kernel_h = input_h + pad_begin[1] + pad_end[1];
+            }
+            if ((input_w > 0) && (input_w + pad_begin[2] + pad_end[2] < kernel_w)) {
+                kernel_w = input_w + pad_begin[2] + pad_end[2];
+            }
+        }
+
+        if (pooling_type == "max") {
+            pool_outputs = std::make_shared<default_opset::MaxPool>(data,
+                                                                    ov::Strides(strides.begin(), strides.end()),
+                                                                    ov::Strides{1, 1, 1},
+                                                                    pad_begin,
+                                                                    pad_end,
+                                                                    ov::Shape{kernel_d, kernel_h, kernel_w},
+                                                                    rounding_type,
+                                                                    auto_pad,
+                                                                    ov::element::i32,
+                                                                    2)
+                               ->outputs();
+        } else {
+            bool exclude_pad = node.get_attribute<bool>("exclusive", false);
+            pool_outputs = std::make_shared<default_opset::AvgPool>(data,
+                                                                    ov::Strides(strides.begin(), strides.end()),
+                                                                    pad_begin,
+                                                                    pad_end,
+                                                                    ov::Shape{kernel_d, kernel_h, kernel_w},
+                                                                    exclude_pad,
+                                                                    rounding_type,
+                                                                    auto_pad)
+                               ->outputs();
+        }
+    }
+
+    if (data_format == "NDHWC") {
+        pool_outputs[0] = std::make_shared<default_opset::Transpose>(
+            pool_outputs[0],
+            std::make_shared<default_opset::Constant>(ov::element::i64, Shape{5}, std::vector<int64_t>{0, 2, 3, 4, 1}));
+    }
+
+    auto output_name = node.get_output_names();
+    if (output_name.size() == 1) {
+        return NamedOutputs{{"Out", {pool_outputs[0]}}};
+    } else {
+        PADDLE_OP_CHECK(node, pool_outputs.size() == 2, "returnmask can`t use with global_pooling");
+        return NamedOutputs{{"Out", {pool_outputs[0]}}, {"Mask", {pool_outputs[1]}}};
```

Comment:
Which case still need to output "Mask" execept 'max_pool3d_with_index'?

Reply:
It seem could be removed, Any return_mask will to `max_pool3d_with_index`